test/box/stream.result

```diff
@@ -0,0 +1,416 @@
+-- test-run result file version 2
+-- This test checks streams iplementation in iproto (gh-5860).
+net_box = require('net.box')
+ | ---
+ | ...
+fiber = require('fiber')
+ | ---
+ | ...
+test_run = require('test_run').new()
+ | ---
+ | ...
+
+test_run:cmd("push filter 'builtin/box/net_box.lua:[0-9]+: " .. \
+             "You can use stream_id, which is generated' " .. \
+             "to 'builtin/box/net_box.lua:<NUM>: " .. \
+             "You can use stream_id, which is generated'")
```

Comment:
You don't need the entire message. The file name + line number are enough. But I now realized you might not need these filters. This diff makes the error message related to the caller, not to netbox internals:
```Diff
@@ -1210,7 +1210,7 @@ function remote_methods:stream(stream_id)
     local last_sid = self._last_stream_id
     if (stream_id and last_sid > 0) or (not stream_id and last_sid < 0) then
         error("You can use stream_id, which is generated automatically, " ..
-              "or set it manually, but do not mix these two options")
+              "or set it manually, but do not mix these two options", 2)
     end
     self._last_stream_id = stream_id and -1 or (last_sid + 1)
     stream_id = stream_id and stream_id or self._last_stream_id
```

Reply:
Fixed

Reply:
box.error(E_PROC_LUA, is better